compile/WINDOWS_INSTRUCTIONS.md

```diff
@@ -0,0 +1,59 @@
+### Steps to compile IRBEM x64 on Windows
+1. Download and install MSYS2 https://www.msys2.org/
+2. Following the installation steps and update the packages:
+```
+pacman -Syu
+```
+After packages are updated, the terminal will be closed.
+3. Run `MSYS2 MSYS` from the start menu and update the rest of the packages
+```
+pacman -Su
+```
+4. Install base-devel package (https://packages.msys2.org/group/base-devel)
+```
+pacman -S base-devel
+```
+5. Install mingw-w64-x86_64-toolchain package (https://packages.msys2.org/group/mingw-w64-x86_64-toolchain)
+```
+pacman -S mingw-w64-x86_64-toolchain
+```
+Note that this is a large package but provides easiest installation process.
+6. Install git package as it required by the IRBEM make file (https://packages.msys2.org/base/git)
+```
+pacman -S git
+```
+7. Close this terminal and open `MSYS2 MinGW x64`.
+8. Navigate to the IRBEM folder. In the following example it is located at c:\libs\IRBEM\
+```
+cd /c
+cd libs/IRBEM
+```
+9. (*) If your **have not cloned IRBEM library** you can do it now using `MSYS2 MinGW x64` terminal:
+```
+cd /c/libs/
+git clone https://github.com/PRBEM/IRBEM.git
+```
+10. Compile the IRBEM library
+```
+make OS=win64 ENV=gfortran64 all
+make OS=win64 ENV=gfortran64 install
+```
+11. Congratulations! Now you have compiled libirbem.dll library in the root folder of IRBEM.
+12. Since libirbem.dll  is a shared library make sure to include `c:\msys64\mingw64\bin\` into Windows Path variable: 
+> Press Win+x->Select "System"->In the "About" section find and open "Advanced system settings"->Click button "Environmental Variables..."->Find variable "Path" and click "Edit..."->Click "New" and include `c:\msys64\mingw64\bin\`-> Close all the windows by clicking "Ok". 
+
+Change `c:\msys64\mingw64\bin\` according to you MSYS2 installation path. 
+
+### How to compile IRBEM x32 on Windows
+The steps are similar as in the previous description, except a few changes:
+
+5. Instead of `mingw-w64-x86_64-toolchain` install package `mingw-w64-i686-toolchain`
```

Comment:
So the fastest way I installed the library as 32-bit was to fire up MobaXTerm (https://mobaxterm.mobatek.net/download.html), install git, gfortran and make with `apt-get install git gcc-fortran make`.
Then I just cloned and ran make with the `OS=win32 ENV=gfortran` options. Rather than change anything here, perhaps we can use the github discussions on the repo and make a post there that documents different ways to install on windows? That way folks can have instructions using MSYS, MobaXTerm, Git Bash, WSL/WSL2, anaconda, etc. and it doesn't need to hide in the `README` files.

Reply:
It is a good suggestion for the future, but for now we need to provide at least one consistent way to compile the library on Windows. I tested it and it is good to go. To keep things simple I suggest to keep it the way it is.